Amplify/Categories/Auth/Result/AuthForgotPasswordResult.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct AuthForgotPasswordResult {
```

Comment:
I would have expected the term Response vs Result, but is suspect we are trying to match some existing api?

Reply:
okay.. do we know how android is handling this?